client/src/pages/FavouriteDestinations/FavoriteDestinantions.js

```diff
@@ -0,0 +1,67 @@
+import React, {useState} from 'react'
+import {Typography, Grid, Button} from "@material-ui/core";
+import places from "../../database/places";
+import Favorite from "@material-ui/icons/Favorite";
+import {CustomSmallerCheckBox} from "../../themes/theme";
+import useStyles from './style'
+
+
+function FavoriteCheckBox() {
+    const classes = useStyles();
+    const [checked, setChecked] = useState(false);
+    return (
+      <div>
+        <CustomSmallerCheckBox
+          checked={checked}
+          onChange={(e) => setChecked(e.target.checked)}
+          icon={<Favorite style={{ color: "white" }} />}
+          checkedIcon={<Favorite style={{ color: "#FFA000" }} />}
+          classes={{ root: classes.customCheckBoxRoot }}
+        />
+      </div>
+    );
+  }
+
+
+function FavoriteDestinantions() {
+
+    const classes = useStyles();
+    return (
+        <Grid className={classes.root}>
+          <Button variant="outlined" style={{float: "right", marginRight: 25}}>Explore</Button>
+          <Typography className={classes.title} variant="h5">Favorite Destinations</Typography>
+            <Grid
+   container
+   spacing={3}
+   justify="center"
+   style={{  marginTop: 10 }} >
+   {places.map((place, index) => (
+     <Grid item key={index}>
```

Comment:
It's usually not a good idea to use the `index` as keys to map over components in React. You could run into [some issues](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318). If the `place` objects don't have unique ids, use something that will always be unique about them.

Reply:
I'll make a push effecting the changes immediately.